### DISCUSSION TITLE
Add icons / status information to room title

### DISCUSSION THREAD

**Discussion by legraml** [matt8707/hass-config#145] [Ideas]
I really like the look'n'feel and layout of this UI, only trouble is showing as much information as possible in a limited view field.

Maybe it would be nice to show additional information e.g. room temperature next to the room header in the button card grid, like a small notifier icon with numbers and the background color changes when heating (red) / cooling (blue) or set temperature is reached (green). Any idea how to get this done quickly?

**Comment by matt8707** [comment]
To keep this config easy to read every section is a grid card. If you want to you can replace that with a layout-card and use a button-card as a title instead.
